src/e2e/registration.spec.js

```diff
@@ -0,0 +1,28 @@
+import { getRegistrationData } from "../utils/registration-data";
+import { UserActions } from "../project/user-actions";
+import { test } from '@playwright/test';
+
+test.describe('Check registration form', () => {
+
+    test.beforeEach(async ({ page }) => {
+        await page.goto('https://qauto.forstudy.space/');
+    });
+
+    test('Register new customer and login', async ({ page }) => {
+        const registrationData = getRegistrationData();
+        const userActions = new UserActions(page);
+
+        await userActions.registerNewCustomer(registrationData);
+        await userActions.logout();
+        await userActions.login(registrationData);
+    });
+
+    test('Check validation errors', async ({ page }) => {
+        const registrationData = getRegistrationData();
+        const userActions = new UserActions(page);
+
+        await userActions.checkValidationErrors(registrationData);
```

Comment:
how many scenarios do you have encapsulated here?

You put a lot of test cases in one method

The most obvious disadvantage that you have dependent tests

if first validation check fails others will not be performed